setup-env/test/actionInput/inputValidator.test.js

```diff
@@ -152,6 +152,29 @@ describe('InputValidator class to validate individual fields of the action input
           expect(InputValidator._getBuildInfo()).to.eq(expectedValue);
         });
       });
+
```

Comment:
Please share prev test coverage vs current for comparision.

Reply:
Added on slack thread

Reply:
Shared on chat